features/mbedtls/targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/mbedtls_device.h

```diff
@@ -20,10 +20,10 @@
 #ifndef MBEDTLS_DEVICE_H
 #define MBEDTLS_DEVICE_H
 
-#define MBEDTLS_AES_ALT
+//#define MBEDTLS_AES_ALT
 
-#define MBEDTLS_SHA1_ALT
+//#define MBEDTLS_SHA1_ALT
```

Comment:
@avolinski 
1. Why removing SHA1 & MD5
2. Why not add a fix to the H/W driver? did you look at the spec maybe its an easy fix

Reply:
SHA found to be messing the attestation test when working with accelerator

Reply:
until the accelerator issue is settled no taking chances for sporadic possible failures